PIM/UserInterface/02a_List.md

```diff
@@ -0,0 +1,1419 @@
+
+- *Destination attribute set for connection*   
+  Destination attribute set for the selected connection. The attribute set is automatically selected. The field is read-only. An own attribute set is assigned to each selected connection.
+
+- [< BACK]   
+  Click this button to go back to the previous step. The *Select Connections and status* wizard window is displayed.
+
+- [CANCEL]   
+  Click this button to cancel adding multi-connection offers and close the *Select destination attribute sets* wizard window.
+
+- [FINALIZE]   
+  Click this button to save the multi-connection offers and close the *Select destination attribute sets* wizard window.
+
+
+
+## Edit product
+*PIM > Products > Tab LIST > Select product*
+
+![Edit Product](../../Assets/Screenshots/PIM/Products/List/EditProduct.png "[Edit Product]")
+
+- ![Back](../../Assets/Icons/Back02.png "[Back]") (Back)   
+  Click this button to close the *Edit product* view and return to the product list. All changes are rejected.
+
+- [CANCEL]   
+  Click this button to cancel editing the product.
+
+- [SAVE]   
+  Click this button to save the changes to the product.
+
+- Image box
+  Image of the product. The image is displayed when an image is assigned in the corresponding attribute.
+
+- *SKU*   
+  Stock Keeping Unit. Identification number for the product.  he SKU (Stock Keeping Unit) is an identification number for the product. Therefore, it should be uniquely assigned to a single product.   
+  Click the button ![Edit](../../Assets/Icons/Edit02.png "[Edit]") (Edit) to the right of the SKU to edit it.
+
+- ![Edit](../../Assets/Icons/Edit02.png "[Edit]") (Edit)   
+  Click this button to edit the SKU.
+
+- ![Apply](../../Assets/Icons/Check.png "[Apply]") (Apply)   
+  Click this button to apply the changes to the SKU. This button is only displayed when you are editing the SKU.
+
+- *Channel*   
+  Click the drop-down list to select a channel for the product. All active channels are displayed in the drop-down list. By default, the **Actindo Basic** channel is preselected.
+
+- *Language*   
+  Click the drop-down list to select a language for the product. All active languages are displayed in the drop-down list. The default language is preselected.
+
+
+- [Complete]     
+  Completeness of the product for all available channels and languages. The percentage of completeness is displayed in the button. Click the button to display the *Completeness* window. If the completeness is less than 100%, the number of missing require attributes is displayed to the right of the completeness button.
+  If the completeness cannot be calculated, *Completeness: Not available* is displayed and the button is not clickable.
+
+  > [Info] The completeness is calculated by considering the number of missing required fields. As the required fields are defined individually for each attribute set set, it can vary from one product to another.  
```

Comment:
```suggestion
  > [Info] The completeness is calculated by considering the number of missing required fields. As the required fields are defined individually for each attribute set configured, it can vary from one product to another.  
```